ibis/backends/tests/test_export.py

```diff
@@ -215,6 +215,20 @@ def test_table_to_parquet(tmp_path, backend, awards_players):
     )
 
 
+def test_table_to_parquet_dir(tmp_path, backend, awards_players):
+    outparquet_dir = tmp_path
```

Comment:
This fails because `tmp_path` is a directory. Check out the usage of it throughout the codebase, and also the [pytest docs on `tmp_path`](https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture) to get an idea for how to use it correctly.